pkgs/top-level/all-packages.nix

```diff
@@ -30026,6 +30026,8 @@ with pkgs;
 
   quattrocento-sans = callPackage ../data/fonts/quattrocento-sans { };
 
+  quinze = callPackage ../data/fonts/quinze { };
+
```

Comment:
```suggestion
```
New top-level packages should go to `pkgs/by-name` https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

Reply:
oh that's nice <3
done